recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/images/core-image-weston.bbappend

```diff
@@ -16,6 +16,8 @@ IMAGE_INSTALL_append = " \
 python __anonymous () {
     if (d.getVar("AOS_VIS_PACKAGE_DIR", True) or "") == "" and not "domu" in (d.getVar("XT_GUESTS_INSTALL", True).split()):
         d.appendVar("IMAGE_INSTALL", "aos-vis")
+    if (d.getVar("AOS_VIS_PACKAGE_DIR", True) or "") != "" and not "domu" in (d.getVar("XT_GUESTS_INSTALL", True).split()):
+        d.appendVar("IMAGE_INSTALL", "ca-certificates")
```

Comment:
I wonder if those two if-s could be consolidated into a single one?

Reply:
Done